xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -32,6 +32,7 @@
 import io.grpc.xds.Filter.NamedFilterConfig;
 import io.grpc.xds.LoadStatsManager2.ClusterDropStats;
 import io.grpc.xds.LoadStatsManager2.ClusterLocalityStats;
+import io.grpc.xds.internal.sds.TlsContextManager;
```

Comment:
Ideally, should `io.grpc.xds.internal` depends on `io.grpc.xds` or the reverse? We already have circular dependencies and had plan to fix it. For new changes, we should try the best to avoid introducing new circular dependency.

Reply:
Good point. I will move TlsContextManager to the public package and leave the *Impl inside internal.

Reply:
done